dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/delete-fee-record-correction-transient-form-data.api-test.ts

```diff
@@ -0,0 +1,88 @@
+import { HttpStatusCode } from 'axios';
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordCorrectionTransientFormDataEntityMockBuilder,
+  FeeRecordEntityMockBuilder,
+  RECONCILIATION_IN_PROGRESS,
+  RECORD_CORRECTION_REASON,
+  RecordCorrectionTransientFormData,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import { withSqlAndMongoIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { testApi } from '../../../test-api';
+import { SqlDbHelper } from '../../../sql-db-helper';
+
+console.error = jest.fn();
+
+const BASE_URL = '/v1/utilisation-reports/:reportId/fee-records/:feeRecordId/correction-transient-form-data/:userId';
+
+describe(`DELETE ${BASE_URL}`, () => {
+  const getUrl = (reportId: number | string, feeRecordId: number | string, userId: string) =>
```

Comment:
is this being used in more than 1 file? if so, maybe a test helper?

Reply:
Pulled out into `dtfs-central-api/test-helpers/replace-url-parameter-placeholders.ts`, replaced all record correction `getUrl` functions with calls to this thanks

Reply:
Added a new ticket `FN-3711` for replacing all other `getUrl` function declarations in the other (non-record-correction) API tests